### PR TITLE
Propose modes

### PR DESCRIPTION
**Pull request by rajwinderb**
These functions will remove "walking" as a viable mode if the distance is greater than 5km. We have also created the function to change the distances out of the google maps API into a float in order to do this.

Note: we have left variables and prints in so you can test this yourself but we will remove this at the end.

### CODE REVIEW

**Comment by Sravya12379** [comment]
Wow!! Great work in suggesting proposed modes!! I just wanted to suggest a small change. I think it would be nice to set threshold distance to Cycling even for super distances case, by looking at super distance figures, I don't feel it's correct to suggest cycling for thousands of kilometres. Of course, we can list modes of transport but in the suggested mode,  may we need to limit cycling to a certain value.  

So, I finally want to suggest that if possible we can list 'available modes of transport and 'proposed modes' side by side maybe

File: propose_mode.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    This function removes the "walking" mode, from the distances dictionary,if the distance of the journey is greater
+    than 5km as will no longer be a viable mode of transport.
+
+    :param distances: distances as dictionary (as would be received from the function get_distance())
+    :return: distances as dictionary only including the viable modes of transport
+    """
+    distances = str_to_float(distances)
+    if distances['walking'] > 5:
+        del distances['walking']
+
```
Comment (robynfsj) [inline propose_mode.py:31]: Should we do the same for cycling? e.g. if > 100km

**Review by robynfsj — APPROVED** [review]

File: propose_mode.py (hunk 1 above)
Comment (rajwinderb) [reply propose_mode.py:33]: Just added it in ☺️

**Review by Laksh-13 — APPROVED** [review]
Runs Perfect!

**Pull request merged**